test/dlc_tests/eks/mxnet/training/test_eks_mxnet_training.py

```diff
@@ -21,7 +22,7 @@ def test_eks_mxnet_single_node_training(mxnet_training):
     pod_name = f"mxnet-single-node-training-{rand_int}"
 
     args = (
-        "git clone https://github.com/apache/incubator-mxnet.git && python "
+        "git clone -b 1.6.0 https://github.com/apache/incubator-mxnet.git && python "
```

Comment:
nit: is there a way we could get this value from the buildspec, to avoid having to manually change this when there are new versions?

Reply:
Getting it from the buildspec might be hard, but it may be possible to recover this information from the DLC tag.